### ISSUE TITLE
Quanergy M8 

### ISSUE THREAD

**Issue by danbo2** [HKUST-Aerial-Robotics/A-LOAM#31]
Hi, if I want to use the data collected from Quanergy M8 Lidar, what should I do?  I find the map can not be generated when I play the data collected with Quanergy M8. Thanks

**Comment by cristianrubioa** [comment]
Hi @danbo2 have you found a way to do it with the Quanergy M8 Lidar?

**Comment by danbo2** [comment]
Yes, you can use node transition method in ROS and I encourage you to learn more about ROS. I just add one line 'rosrun topic_tools throttle messages /Sensor/points 10 /velodyne_points'. It means I convert /Sensor/point 10 to /velodyne points.

**Comment by cristianrubioa** [comment]
Thanks @danbo2 I can already generate the map. I can ask you what additional parameters you set for this Lidar in the algorithm?

**Comment by danbo2** [comment]
I don't think I changed any parameters.

**Comment by cristianrubioa** [comment]
I mean adapt-> https://github.com/HKUST-Aerial-Robotics/A-LOAM/blob/devel/src/scanRegistration.cpp for this specific QM8 Lidar

**Comment by danbo2** [comment]
No, I didn't make any change of the original source code.
